elide-core/src/main/java/com/yahoo/elide/core/EntityPermissions.java

```diff
@@ -101,6 +104,9 @@ private void bindMemberPermissions(Map<String, ParseTree> fieldPermissions,
 
     private ParseTree getPermissionExpressionTree(Class<? extends Annotation> annotationClass, Annotation annotation) {
         try {
+            if (annotationClass == SharePermission.class) {
+                return null;
```

Comment:
This should never happen, should it? `ReadPermission` should now be evaluated instead?